common/types/shared.go

```diff
@@ -8174,3 +8174,6 @@ func SerializeRequest(request interface{}) (string, error) {
 
 	return string(res), nil
 }
+
+// ContextKey is for Pinot/ES response comparator. This struct will be passed into ctx as a key.
+type ContextKey string
```

Comment:
This name is unfortunately a bit too general for use for the Pinot / Comparitor. Not entirely sure what you're aiming for sorry. if we need to create a type, maybe we can either prefix it with the `VisibliityManager` and maybe keep it in Persistence? The types here are intended for IDL mapping only

Reply:
moved it to a different folder